bin/etc-update

```diff
@@ -32,7 +32,7 @@ get_config() {
 		"${PORTAGE_CONFIGROOT}"etc/etc-update.conf)
 }
 
-OS_RELEASE_ID=$(cat /etc/os-release 2>/dev/null | grep '^ID=' | cut -d'=' -f2 | sed -e 's/"//g')
+OS_RELEASE_ID=$(source /etc/os-release >/dev/null 2>&1; echo $ID)
```

Comment:
Just in case $ID contains special shell characters, we should use `echo "$ID"`.

Reply:
Ok then the `OS_RELEASE_ID` variable is not really necessary and we could simply use `ID`

Reply:
Well, the isolation provided by the subshell is useful, since lots of things could go wrong when a file is sourced.

Reply:
yes sure, agreed.

Reply:
Also `${ID}`, please, for consistency with what we tend to use everywhere.

Reply:
When echoing a variable, shell characters in the variable definition shouldn't be reinterpreted? Quotes should only really matter for word splitting, which as a matter of good practice should still be quoted on those grounds alone.

The actual practical effect would be that 
```
$ ID="foo    bar"
$ echo $ID
foo bar
$ echo "$ID"
foo    bar
$ IFS=f
$ echo $ID
 oo    bar
$ echo "$ID"
foo    bar
```
Failure to quote, means the shell collapses whitespace (usually) due to reinterpreting it by way of `$IFS`.